chainerx_cc/chainerx/stack_vector.h

```diff
@@ -194,4 +195,33 @@ class StackVector {
     BaseContainer d_{};
 };
 
+// Formatter to print StackVector containing integral elements as e.g. '[]' or '[1, 2, 3]'.
```

Comment:
Do not implement here: this is not a feature of `StackVector`.

Reply:
How moving it to a new `chainerx_cc/chainerx/stack_vector_formatter.h`?

Reply:
Fixed.